### ISSUE TITLE
audio note not synced to nextcloud 

### ISSUE THREAD

**Issue by thri1hous** [quillpad/quillpad#226]
i understand nextcloud syncing is considered experimental. thought id share results of my "experimenting"

if i create an audio note in quillpad then sync to nextcloud, i can see the note in nextcloud and read the markdown associated with that note but the actual audio attachment doesnt seem to be synced

**Comment by mapausch** [comment]
similar situation here: images in notes are not synced to Nextcoud.

**Comment by zzyzx-dc** [comment]
Yeah I ran into this recently. I think I reinstalled Quillpad and my image notes just had empty images - blank, but the image filenames were still there. I suppose they were only saved locally even though the text portion and the file were saved to the Nextcloud API. (No big deal since I have copies)

**Comment by dudumaroja** [comment]
It's written on the Nextcloud setting menu that attachments are not supported for now, I hope this change since nextcloud notes does support mídia attachments now, the devs just need to add  it
